Cargo.toml

```
@@ -15,4 +15,5 @@ build = "build.rs"
 leptonica-sys = "0.3.0"
 
 [build-dependencies]
-bindgen = "0.52"
\ No newline at end of file
+bindgen = "0.52"
+vcpkg = "0.2.8"
```

It looks like this can be specified for just Windows
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies

```suggestion
[target.'cfg(windows)'.build-dependencies]
vcpkg = "0.2.8"
```

---

Yes, I was going to figure out how to do it but forgot. Thanks for finding!